conda_build/skeletons/cpan.py

```diff
@@ -289,21 +302,21 @@ def skeletonize(packages, output_dir=".", version=None,
                                                'import_tests': ''})
 
         # Fetch all metadata from CPAN
-        if version is None:
+        core_version = core_module_version(package, perl_version, config=config)
+        if version in [None, ''] and core_version is None:
```

Comment:
In Python you can just test the variable and it will already test if it is ``None`` or empty. Such as:
```suggestion
        if not version and not core_version:
```

Reply:
Good point.  It was this way in the old code and I agree we should clean it up.